src/index.ts

```diff
@@ -43,6 +43,8 @@ async function run() {
     if (githubUserEmail) {
       await exec.exec(`git config --global user.email "${githubUserEmail}"`);
     }
+
+    await exec.exec(`git fetch origin ${headRef} ${baseRef}`);
```

Comment:
Let's leave explicit comment here that this is required for V2 with link to documentation, explaining WHY

Reply:
There is no documentation on HOW `actions/checkout` works and `git fetch` is a obvious command that shouldn't be a problem to anybody

Reply:
Let's least add a comment that it's required for checkout v2